src/throttler/throttler.ts

```diff
@@ -75,6 +74,18 @@ export class Queue<T> {
     }
   }
 
+  /**
+   *
+   * @return {boolean} true if there are unscheduled task waiting to be scheduled.
+   */
+  public abstract hasWaitingTask(): boolean;
+
+  /**
+   *
```

Comment:
Remove the extra empty comment line (that contains only a single `*`) - same above